common/buildcraft/core/BCCoreConfig.java

```diff
@@ -196,6 +197,12 @@ public static void preInit(File cfgFolder) {
         propNetworkUpdateRate.setComment(
             "How often, in ticks, should network update packets be sent? Increasing this might help network performance.");
         none.setTo(propNetworkUpdateRate);
+        
+        propWorldMin = config.get(general, "worldMin", 0);
+        propWorldMin.setMinValue(-2147483648).setMaxValue(2147483647);
+        propWorldMin
+            .setComment("How deep is the Minecraft world? (Vanilla = 0, default = 0, Cubic Chuncks = -1073741824)");
```

Comment:
This seems a little... odd. Why not use a switch that detects if Cubic Chunks is enabled/loaded and changes it accordingly? ATM there is no reason to set this between 0 and -1073741824.